### PR TITLE
comparison table and feature-box fix for landings

### PR DESCRIPTION
**Pull request by Ayhanemoon**

### CODE REVIEW

File: src/components/Widgets/ComparisonTable/ComparisonTable.vue [hunk 1]
```diff
@@ -73,74 +73,64 @@ export default {
...
-      // })
-      for (let index = 0; index < this.header.length; index++) {
-        const header = this.header[index]
-        this.columns.push(
+      for (let index = 0; index < this.localOptions.header.length; index++) {
+        if (this.localOptions.columns.some(e => e.name === `col${index}`)) {
+          return
+        }
+        const header = this.localOptions.header[index]
+        this.localOptions.columns.push(
```
Comment (kerasus) [inline src/components/Widgets/ComparisonTable/ComparisonTable.vue:106]: get column from computed

File: src/components/Widgets/ComparisonTable/ComparisonTable.vue [hunk 2]
```diff
@@ -73,74 +73,64 @@ export default {
...
-    },
     getColumns() {
-      // this.columns.push({ name: 'attribute', label: this.options.attributesLabel, field: 'attribute' })
-      // this.records.forEach(record => {
-      //   this.columns.push({ name: record.key, label: record.title, field: record.key })
-      // })
-      for (let index = 0; index < this.header.length; index++) {
-        const header = this.header[index]
-        this.columns.push(
+      for (let index = 0; index < this.localOptions.header.length; index++) {
```
Comment (kerasus) [inline src/components/Widgets/ComparisonTable/ComparisonTable.vue:101]: use const variable for this.localOptions.header.length

File: src/components/Widgets/ComparisonTable/ComparisonTable.vue [hunk 3]
```diff
@@ -73,74 +73,64 @@ export default {
...
+        flat: false
+      }
     }
   },
   watch: {
-    options: {
-      handler() {
-        this.loadConfig()
-      }
+    options() {
```
Comment (kerasus) [inline src/components/Widgets/ComparisonTable/ComparisonTable.vue:90]: remove this

File: src/components/Widgets/ComparisonTable/ComparisonTable.vue [hunk 4]
```diff
@@ -73,74 +73,64 @@ export default {
...
+      this.getColumns()
+      // this.getRows()
     }
   },
   mounted() {
-    this.loadConfig()
+    this.getColumns()
```
Comment (kerasus) [inline src/components/Widgets/ComparisonTable/ComparisonTable.vue:96]: remove this

File: src/components/Widgets/ComparisonTable/ComparisonTable.vue [hunk 5]
```diff
@@ -73,74 +73,64 @@ export default {
...
+        this.localOptions.columns.push(
           {
             name: `col${index}`,
             label: header
           })
       }
     },
     getRows() {
-      this.attributes.forEach(attribute => {
+      this.localOptions.attributes.forEach(attribute => {
```
Comment (kerasus) [inline src/components/Widgets/ComparisonTable/ComparisonTable.vue:114]: remove getRows()

Comment (Ayhanemoon) [reply src/components/Widgets/ComparisonTable/ComparisonTable.vue:114]: removed

**Review by kerasus — APPROVED** [review]

**Pull request merged**